pkg/printers/humanreadable.go

```diff
@@ -399,7 +406,14 @@ func PrintTable(table *metav1beta1.Table, output io.Writer, options PrintOptions
 // does not expose metadata). It returns an error if the table cannot
 // be decorated.
 func DecorateTable(table *metav1beta1.Table, options PrintOptions) error {
```

Comment:
from https://github.com/kubernetes/kubernetes/pull/72418#discussion_r244656695
> @smarterclayton:
> Kind of yucky. A table should know its column structure, and an event should be <underlying printer> + event type wrapper, so I feel like this shouldn't be necessary.

Reply:
left table output alone for now... will let you resolve that in your table output watch PR